tests/sles4sap/saptune/mr_test.pm

```diff
@@ -23,7 +23,7 @@ sub reboot_wait {
     $self->reboot;
 
     # Wait for tuned to tune everything
-    my $timeout = 60;
+    my $timeout = bmwqemu::scale_timeout(60);
```

Comment:
Not good here, as the `sleep` for non sle15 will be scale twice and `assert_script_run` also already handle the scale.

And in fact `assert_script_run` will kill the shell command when the timeout is reached, so the `timeout` command inside the shell is not needed I think (you may test, sorry for didn't saw this the last time). So `assert_script_run "bash -c 'until tuned-adm verify >/dev/null ; do sleep 1 ; done'", $timeout` should be the only change here.

Reply:
Good catch!

Fixed.